cleanlab/classification.py

```diff
@@ -370,6 +371,11 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
           This can be useful for training differently in the final ``fit()``
           than during cross-validation.
 
+        validation_func : callable, optional
+          Optional callable function that takes two arguments, `X_val`, `y_val`.
+          Specifies how to map the validation data split in the cross validation
+          step into the appropriate format to pass into `clf`'s ``fit()`` method.
+
```

Comment:
provide a simple example?  

Example: If your model's fit function looks like this: ``clf.fit(X,y,X_validation,y_validation)``, then you could set: ``validation_func = f``, with: ``def f(X_val, y_val): return {"X_validation": X_val, "y_validation": y_val}``

Reply:
Add final statement to docstring:

Note that ``validation_func`` will be ignored in the final call to `clf.fit()` on the cleaned subset of the data. This argument is only for allowing `clf` to access the validation data in each cross-validation fold (eg. for early-stopping or hyperparameter-selection purposes). If you want to pass in validation data even in the final training of `clf.fit()` on the cleaned data subset, you should explicitly pass in that data yourself (eg. via ``clf_final_kwargs`` or ``clf_kwargs``).

Reply:
done!